### DISCUSSION TITLE
How do I make the TitleBar transparent?

### DISCUSSION THREAD

**Discussion by eduardobragaxz** [microsoft/microsoft-ui-xaml#8295] [Q&A] [answered]
I'm trying to mimic what I see in the WinUI Gallery and Clock apps, which is the titlebar showing in all DisplayModes and transparent. I got to make it to kind of work by copying some code from the WinUI Gallery repo, but the TitleBar does not become transparent.

[Here's](https://github.com/eduardobragaxz/Tests/blob/b467ea4b82aab397260adbac00b43725d4ccbd8a/MainPage.xaml) the MainPage.xaml file
And [here's](https://github.com/eduardobragaxz/Tests/blob/a6251583601908ac517597429fb3e61126b63f3f/MainPage.xaml.cs) the code behind.
The only thing in the MainWindow files are a Page in XAML setting its content to MainPage in the code behind.

In App.xaml.cs I have a static property
`Window MWindow = {get => m_window; set => m_window = value; }`

Please ignore the MainWindow files in the repo from the links I put here. It's for another issue.

I can make the TitleBar transparent if I set the AppWindow.ExtendsContentIntoTitleBar to true, but then I don't know how to put anything in it.

Here are some screenshots of what I'm trying to accomplish
![expanded](https://user-images.githubusercontent.com/50815710/225933366-4780745b-608a-4563-bda6-ab244dd7be3e.png)
![minimal](https://user-images.githubusercontent.com/50815710/225933471-9facfae3-002a-4464-a2cc-60644611475b.png)
![expanded](https://user-images.githubusercontent.com/50815710/225934266-c997421a-7df0-4171-870a-3b8658cfbbda.png)
![minimal](https://user-images.githubusercontent.com/50815710/225934334-5ead3a70-8438-4788-ba28-e023a1b57749.png)

Here's my failed attempt
![fail](https://user-images.githubusercontent.com/50815710/225933651-086c4512-388f-4c19-8fcf-28b676ee9d09.png)

Any help is appreciated.

**Comment by TRadigk** [comment, marked as answer]
Hi @eduardobragaxz

to achieve this result:
![grafik](https://user-images.githubusercontent.com/92845708/226088587-39886ade-7e79-456f-8ca5-b2d70ce60fd0.png)

please add this line in app.xaml to the key **ResourceDictionary:**
`<SolidColorBrush x:Key="WindowCaptionBackground">Transparent</SolidColorBrush>`

On your xaml for the page set <Border> **Background** to **Transparent.**

**Reply by eduardobragaxz** [reply]
Thanks @TRadigk. It does work when the window has focus, but if I focus somewhere else it looks like it does in my screenshot, not sure why.

(It looks like you tagged the wrong person hahaha)

**Reply by TRadigk** [reply]
Oh, I assumed github was smart enough to pick the correct "eduardo".

If you haven't figured it out yourself, add this line to the app.xaml resource dictionary:
`<SolidColorBrush x:Key="WindowCaptionBackgroundDisabled">Transparent</SolidColorBrush>`

**Reply by eduardobragaxz** [reply]
That's it. Thank you!
